src/hotspot/share/gc/epsilon/epsilonBarrierSet.hpp

```diff
@@ -1,4 +1,5 @@
 /*
+ * Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Why the copyright line addition, though? It is not like Red Hat adds its copyrights for trivial changes to shared files like these.

Reply:
Removed.